tests/components/test_helm_app.py

```diff
@@ -160,7 +163,7 @@ async def test_should_deploy_app_with_local_helm_chart(
         app_values: HelmAppValues,
     ):
         class AppWithLocalChart(HelmApp):
-            repo_config: None = None
+            repo_config: HelmRepoConfig | None = None
```

Comment:
judging by the class name, I believe it was set to `None` on purpose

Reply:
I was getting an error on PyRight here. `repo_config` type is `HelmRepoConfig | None`
https://github.com/bakdata/kpops/blob/2edbb7b5aae959df3d954bb79fc9f399c41007a3/kpops/components/base_components/helm_app.py#L64

Reply:
could you ignore the error for now, please? otherwise the override doesn't make sense